tools/target.py

```diff
@@ -21,21 +21,19 @@ def register(command):
         "--build",
         dest="build",
         action="store_true",
-        default=False,
         help="build the image locally, instead of pulling and tagging it",
     )
 
     command.add_argument(
```

Comment:
Nice comments down further. Would be super cool to have similar comments up here.